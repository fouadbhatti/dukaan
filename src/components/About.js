import styled from 'styled-components';
import React from 'react';

const Wrapper = styled.div`
  margin: 100px;
`;

const A = styled.a ``;

const About = () => (
  <Wrapper>
    Github: <A href="https://github.com/khizar-ali/store" target="_blank">https://github.com/khizar-ali/store</A>
  </Wrapper>
);

export default About;
