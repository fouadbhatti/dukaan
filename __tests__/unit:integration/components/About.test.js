import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import About from '../../../src/components/About';

describe('<About />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<About />);
  });

  it('should render without crashing', () => {
    expect(wrapper.length).to.equal(1);
  });

  it('should match snapshot', () => {
    const component = toJson(wrapper);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
