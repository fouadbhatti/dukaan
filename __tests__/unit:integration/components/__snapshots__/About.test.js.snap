// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should match snapshot 1`] = `
<styled.div>
  Github: 
  <styled.a
    href="https://github.com/khizar-ali/store"
    target="_blank"
  >
    https://github.com/khizar-ali/store
  </styled.a>
</styled.div>
`;
